docs/03-TechnicalInfrastructure.md

```diff
@@ -4,13 +4,13 @@
 ## 3.1 Back end directory structure
 
 The root directory of the imaging part of a LORIS instance is typically 
-  `/data/project`.
+  `/data/$PROJECT`.
 
 ```
 ## Imaging pipeline file directory structure
 /
 |__ data
-    |__ project
+    |__ $PROJECT
         |__ bin
         |   |__ mri
         |__ data
```

Comment:
I'm not sure if this is accurate. For my sandbox, and our own project, the `incoming` directory is not nested within `/data/$PROJECT/data` but rather just under `/data/`. 
When using `imaging_install.sh`, by default, the `pipelines`, `protocols` subdirectories are not created under `/data/$PROJECT/data/` either. Maybe we should add an `optional` tag or something?

Reply:
agree on both points. Corrected now and along with your suggested correction, I reordered the directories to be in synch with the order the installer does them, and I added the missing `batch_output` one.